retrying.py

```diff
@@ -268,6 +271,7 @@ def call(self, fn, *args, **kwargs):
                     raise RetryError(attempt)
             else:
                 sleep = self.wait(attempt_number, delay_since_first_attempt_ms)
+                self._logger.info(f"Retrying in {sleep / 1000.0:.2f} seconds.")
```

Comment:
I see your point. Not sure how to get the actual exact wait time, though, since it is actually random. Perhaps just add the jitter range in the log message?

```python
jitter = f"+ (0 to {self._wait_jitter_max})" if self._wait_jitter_max else ""
self._logger. info(f"Retrying in {sleep / 1000.0:.2f} {jitter}seconds.")
```